services/datamanager/builtin/disk_summary_logger_posix.go

```diff
@@ -0,0 +1,9 @@
+//go:build linux || darwin
```

Comment:
why is https://github.com/viamrobotics/rdk/pull/4603/files#diff-a2d8aee5191780089f953177c16340e95bea7525c80ba38ad69f4a2e767de098R1 
```
//go:build !windows
```
while this is `//go:build linux || darwin`?

Reply:
no reason -- switched it to `!windows` in 685175fd3